src/client/api/quoteBundleErrorSelectors.ts

```diff
@@ -75,3 +76,12 @@ export function isLimitHit(
 
   return false
 }
+
+export function getLimitsHitFromEditQuoteMutation(
+  editQuoteMutation: EditBundledQuoteMutation | null | undefined,
+) {
+  if (isQuoteBundleError(editQuoteMutation?.quoteCart_editQuote))
+    return editQuoteMutation?.quoteCart_editQuote.limits || []
+
```

Comment:
nit:

```suggestion
  if (isQuoteBundleError(editQuoteMutation?.quoteCart_editQuote)) {
    return editQuoteMutation?.quoteCart_editQuote.limits || []
  }

```

Reply:
The suggestion and the actual code are the same, no?! 🤔

Reply:
OMG, hahhaha. That's true. Thank you

Reply:
🤣 haha yeah it took me a while as well